istio/install/README.md

```diff
@@ -0,0 +1,16 @@
+# Istio manifest for Kubeflow
+
+The manifest is taken from OSS Istio release 1.0.2.
+
+We need custom configuration as discussed in the
+[issue](https://github.com/kubeflow/kubeflow/issues/1909#issuecomment-438409215).
+
+Specifically, the things we changed are:
+
+1. The policy of configmap `istio-sidecar-injector` is changed from `enabled` to `disabled`.
```

Comment:
I thought Istio injector is disabled by default. IIUC, their suggested pattern is to enable it by namespace. E.G., you make a mesh with a gateway per namespace.

Reply:
The policy is enabled by default in the configmap.
You can check istio's original manifest.

Enabled doesn't mean it's always injecting, it's a combination of this enabled/disabled and namespace label to decide.